src/core/webidl.js

```diff
@@ -184,7 +180,7 @@ function defineIdlName(escaped, data, parent) {
     const styledName = data.type === "operation" ? `${name}()` : name;
     const ofParent = parentName ? ` \`${parentName}\`'s` : "";
     const msg = `Missing \`<dfn>\` for${ofParent} \`${styledName}\` ${data.type}. [More info](https://github.com/w3c/respec/wiki/WebIDL-thing-is-not-defined).`;
-    showInlineWarning(unlinkedAnchor, msg, "");
+    pub("warn", new RsError(msg, name, { elements: [unlinkedAnchor] }));
```

Comment:
yay!

Reply:
There is a subtle bug here :smile: 
`name` (plugin name) is shadowed by `name` in this `defineIdlName` function